.meta/sinks/aws_kinesis_firehose.toml.erb

```diff
@@ -52,7 +52,7 @@ write_to_description = "[Amazon Web Service's Kinesis Data Firehose][urls.aws_ki
 
 <%= render("_partials/fields/_encoding_options.toml",
   namespace: "sinks.aws_kinesis_firehose.options",
-  encodings: ["json", "text"]
+  encodings: ["ndjson", "text"]
```

Comment:
Related to #3849, would you imagine this would actually put a newline at the end of each record?

Reply:
I agree, these are not "newline-delimited JSON". There is no delimiter for sinks like this that take individual messages as part of their API.